### PR TITLE
VP-4688: Extend Contact with 'Title' field

### PR DESCRIPTION
**Pull request by trueboroda**
Allow editing and getting

### CODE REVIEW

**Review by OlegoO — DISMISSED** [review]

File: VirtoCommerce.Storefront.Model/Customer/Contact.cs [hunk 1]
```diff
@@ -27,6 +27,8 @@ public partial class Contact : Member
         public string TimeZone { get; set; }
         public string DefaultLanguage { get; set; }
```
Comment (pushnitsa) [inline VirtoCommerce.Storefront.Model/Customer/Contact.cs]: Remove empty line

File: VirtoCommerce.Storefront.Model/Security/UserRegistration.cs [hunk 2]
```diff
@@ -45,7 +45,9 @@ public partial class UserRegistration
         [FromForm(Name = "customer[birthDate]")]
         public DateTime? BirthDate { get; set; }
         [FromForm(Name = "customer[timeZone]")]
-        public string TimeZone { get; set; }     
-      
+        public string TimeZone { get; set; }
+
```
Comment (pushnitsa) [inline VirtoCommerce.Storefront.Model/Security/UserRegistration.cs]: Remove empty line

File: VirtoCommerce.Storefront/Domain/Customer/MemberConverter.cs [hunk 3]
```diff
@@ -147,7 +148,8 @@ public static Contact ToContact(this customerDto.Contact contactDto)
                 OrganizationId = contactDto.Organizations?.FirstOrDefault(),
                 OrganizationsIds = contactDto.Organizations,
                 Salutation = contactDto.Salutation,
-                PhotoUrl = contactDto.PhotoUrl
+                PhotoUrl = contactDto.PhotoUrl,
+                Title = contactDto.Title
```
Comment (pushnitsa) [inline VirtoCommerce.Storefront/Domain/Customer/MemberConverter.cs]: Add comma

File: VirtoCommerce.Storefront/Domain/Customer/MemberConverter.cs [hunk 4]
```diff
@@ -113,7 +113,8 @@ public static Contact ToContact(this UserRegistration userRegistration)
                 FirstName = userRegistration.FirstName,
                 LastName = userRegistration.LastName,
                 Salutation = userRegistration.Salutation,
-                PhotoUrl = userRegistration.PhotoUrl
+                PhotoUrl = userRegistration.PhotoUrl,
+                Title = userRegistration.Title
```
Comment (pushnitsa) [inline VirtoCommerce.Storefront/Domain/Customer/MemberConverter.cs]: Add comma

**Review by pushnitsa — CHANGES_REQUESTED** [review]

**Review by pushnitsa — APPROVED** [review]

**Pull request merged**